scripts/dap-ami-bootstrap.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -e
+
+amazon-linux-extras install -y docker
+systemctl enable docker
```

Comment:
Are we installing docker as root?

Reply:
There are tests on this repo you can explore to answer such questions.

https://github.com/conjurinc/appliance-docker-ami/pull/17/files/e2f90b837024735d8370b5c16f573ce242fa785c#diff-91fbf9969affc384b6a4613d18156451R55